src/core/resumes/resume.service.ts

```
@@ -3,46 +3,123 @@ import { Injectable } from '@nestjs/common';
-            await this.resumeRepository.getBestResumes(query);
+
+        const { offset = 0, limit = 10 } = query;
+
+        const twoWeeksAgo: Date = new Date;
+        twoWeeksAgo.setDate(twoWeeksAgo.getDate() - 14);
+
+        const resumes = await this.prisma.resume.findMany({
+            where: {
+                isDeleted: false,
+                createdAt: {
+                gte: twoWeeksAgo,
```

ESLint

[error] Insert `····`

(prettier/prettier)